tasks/gen-pki.yml

```diff
@@ -0,0 +1,131 @@
+---
+- name: gen-pki | Check ca key
```

Comment:
Does the ca key exist solely on the first master?  Are we concerned with a single point of failure?

Reply:
No it gets copied out during the distribute task later on.  That being said, rotating the CA key should not be too bad.